### PR TITLE
Case 21691: Improve nitpick web page.

### PR DESCRIPTION
**Pull request by NissimHadar**
https://highfidelity.manuscript.com/f/cases/21691/

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18113/HighFidelity-Beta-PR15175-b35b768.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18113/HighFidelity-Beta-Interface-PR15175-b35b768.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18113/HighFidelity-Beta-PR15175-b35b768.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18113/HighFidelity-Quest-Beta-PR15175-b35b768.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18113/HighFidelity-Beta-PR15175-b35b768.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18113/HighFidelity-Beta-Interface-PR15175-b35b768.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18146/HighFidelity-Beta-PR15175-6c9c58c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18146/HighFidelity-Beta-Interface-PR15175-6c9c58c.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18146/HighFidelity-Beta-PR15175-6c9c58c.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18146/HighFidelity-Quest-Beta-PR15175-6c9c58c.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18146/HighFidelity-Beta-PR15175-6c9c58c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18146/HighFidelity-Beta-Interface-PR15175-6c9c58c.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18250/HighFidelity-Beta-PR15175-51ab888.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18250/HighFidelity-Beta-Interface-PR15175-51ab888.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18250/HighFidelity-Beta-PR15175-51ab888.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18250/HighFidelity-Quest-Beta-PR15175-51ab888.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18250/HighFidelity-Beta-PR15175-51ab888.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18250/HighFidelity-Beta-Interface-PR15175-51ab888.exe

File: tools/nitpick/src/AWSInterface.cpp [hunk 1]
```diff
@@ -53,6 +56,13 @@ void AWSInterface::createWebPageFromResults(const QString& testResults,
     _urlLineEdit = urlLineEdit;
     _urlLineEdit->setEnabled(false);
 
+
+    _urlLineEdit = urlLineEdit;
+    _urlLineEdit->setEnabled(false);
```
Comment (AndrewMeadows) [inline tools/nitpick/src/AWSInterface.cpp]: Lines 60-61 duplicate lines 56-57.  Do we need two copies of those operations?

Comment (NissimHadar) [reply tools/nitpick/src/AWSInterface.cpp]: Corrected.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18280/HighFidelity-Beta-PR15175-b12a268.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18280/HighFidelity-Beta-Interface-PR15175-b12a268.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18280/HighFidelity-Beta-PR15175-b12a268.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18280/HighFidelity-Quest-Beta-PR15175-b12a268.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18280/HighFidelity-Beta-PR15175-b12a268.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18280/HighFidelity-Beta-Interface-PR15175-b12a268.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18300/HighFidelity-Beta-PR15175-1f0fe7d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18300/HighFidelity-Beta-Interface-PR15175-1f0fe7d.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18300/HighFidelity-Beta-PR15175-1f0fe7d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18300/HighFidelity-Quest-Beta-PR15175-1f0fe7d.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18300/HighFidelity-Beta-PR15175-1f0fe7d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18300/HighFidelity-Beta-Interface-PR15175-1f0fe7d.exe

**Pull request merged**